io.github.dbchoco.Salawat.yml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.dbchoco.Salawat
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: salawat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
```

Comment:
does it really need network access?

Reply:
yes, it gets the localisation from the internet

Reply:
can't it get this at build time instead? (mean downloaded prior to the build)

Reply:
I also use the network to look for updates so that the user knows when there is an update, I don't think it's optimal to remove internet access